log/lvl.go

```diff
@@ -64,6 +64,7 @@ var stdOut io.Writer
 var stdErr io.Writer
 
 func init() {
+	ParseEnv()
```

Comment:
Is this new call related to this issue (make ProtocolRegister available to service #656)?